src/modules/snake/snake.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * This is your main app file. Please refer to the documentation for more information.
+ */
+
+/**
+ * If you need to import extra node_modules, use "npm install xxx --save" and place them there.
+ */
+
+var Promise = require("bluebird");
+var _ = require('@qwant/front-i18n')._;
+
+module.exports = {
+
+    /**
+     * (NEEDED)
+     *	This function uses 3 parameters :
+     *	- values : This is an array of values caught by regex.
+     *	For example, if you use the keyword "test" with the trigger "start", and you type "test working?",
+     *  values would be like this :
+     *  	* values[0] = "test working?"
+     *  	* values[1] = "test"
+     *  	* values[2] = "working?"
+     *  But, if you use the trigger "strict", there will be only one value in this array (values[0] = "test working?")
+     *  - proxyURL : If you need to call an external API, use the package "request" with proxyURL as value for
+     *  "proxy" attribute (you can refer to weather IA to check how to use it properly)
+     *  - language : Current language called
+     * @returns data to be displayed.
+     */
+
+    getData: function (values, proxyURL, language) {
+        return new Promise(function (resolve, reject) {
+            resolve("hello");
```

Comment:
Could we set some data here (that are gonna be then used and changed client-side)? Or resolve with something else than "hello"?